profiles/views.py

```diff
@@ -96,19 +101,21 @@ def issue(request):
         for label in labels:
             if label["name"] == 'good first issue':
                 leaderboard.good_first_issue = True
+                leaderboard.points += gfi_points
                 if leaderboard.medium_issues_solved >= 2:
                     leaderboard.milestone_achieved = True
-                leaderboard.save()
             elif label["name"] == 'medium':
                 leaderboard.medium_issues_solved += 1
+                leaderboard.points += medium_issue_points
                 if(leaderboard.good_first_issue and leaderboard.medium_issues_solved == 2):
                     leaderboard.milestone_achieved = True
-                leaderboard.save()
             elif label["name"] == 'hard':
                 leaderboard.hard_issues_solved += 1
-                leaderboard.save()
+                leaderboard.points += hard_issue_points
             else:
                 pass
+            
+            leaderboard.save()
```

Comment:
By indentation I suppose this is inside the `for label in labels:` loop, but leaderboard doesn't need to be saved for every label, but only when there is a change in values. Please crosscheck this.

Reply:
This line was being repeated in most of the conditions, that is why I added it here. I will revert the changes. :disappointed:

Reply:
😅😓